### PR TITLE
[55] add document for database

### PR DESCRIPTION
**Pull request by paradisuman**
## Why you open this Pull Request?

add evaluation for database

## What work have you done in the current Pull Request?

- [x] add 55.tm about database
- [x] add evaluation for database

### CODE REVIEW

File: devel/55.tm [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  Invariant Sections, with no Front-Cover Texts, and with no Back-Cover
+  Texts. A copy of the license is included in the section entitled "GNU Free
+  Documentation License".>
+</body>
+
+<\initial>
+  <\collection>
+    <associate|page-medium|paper>
+  </collection>
+</initial>
```
Comment (Charonxin) [inline devel/55.tm:28]: add a blank line

**Review by Charonxin — APPROVED** [review]

**Comment by Charonxin** [comment]
title should be [55] ...

**Comment by darcy-shen** [comment]
写一些这个数据库的单元测试，然后再补充这个文档

**Comment by darcy-shen** [comment]
由于文档不影响代码，所以我直接merge了

**Pull request merged**